en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
 
-_nanDECK_ has a very useful ability to calculate the items in a spreadsheet of a designated value. It does this using an expression that is designated by curly brakets. In this case, _nanDECK_ translates the expression {(IMAGES)} to a caculation of the number of items with a field of an IMAGE. In our example, there are 58 images and so {(IMAGES)} equals 58. Knowing this, we can now understand that this line of script instructs nanDECK to print the images from rows 1 through 58.
+nanDECK can very usefully calculate the number of items listed under a designated field in a spreadsheet. It does this using an expression delimited by curly brakets. In this case, the expression {(IMAGES)} instructs nanDECK to caculate the number of items listed under the `IMAGE` field. Here, there are 58 images, so {(IMAGES)} returns `58`. Knowing this, you'll see that at line 6 of Figure 5 above, `IMAGE="1-{(IMAGES)}"` instructs nanDECK to print the images from rows 1 through 58.
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-6.png" alt="A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption="Figure 6: A card generated using nanDeck." %}
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-6.png" alt="A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption="Figure 6. A card generated using nanDeck." %}
 
-To generate the card above, _nanDECK_ was directed to insert the image found at _Images/Sandwich_First.jpg_ at the 0,0 position and for the image to take up 100% of the width (if possible) and only 60% of the card’s height (if possible). By using the P flag, _nanDECK_ is asked to calculate where the image should be positioned while maintaining the original image’s proportions. Since there are a variety of images in this deck that are of differing proportions, I opted to make all the images grayscale using the G tag to give the cards in the deck some uniformity and cohesion.
+To generate the card above, nanDECK was directed to insert the image found at `Images/Sandwich_First.jpg`, to place it at the 0,0 position, and make the image take up 100% of the card's width (if possible), but only 60% of the height (if possible). The `P` flag asks nanDECK to maintain the image’s original proportions. I opted to make all the images grayscale using the `G` tag, to give the cards in the deck some uniformity and cohesion.
```

Comment:
I'm a little confused by the use of both 'flag' and 'tag' here – are they different things, or should one of them be changed to match the other?

Reply:
Both instances should be 'flag' as this is the language the developer uses in the manual.  Perhaps this text would be more clear if it referred back to Figure 5... like, "as illustrated in Figure 5, by selecting the checkbox beside 'P', the flag tells nanDECK to maintain the image's original proportions.